R/tokenize.R

```diff
@@ -476,13 +454,15 @@ tokenize_bert <- function(...,
 
   # The return process is the same from here on out regardless of which method
   # was used, so we call a function to deal with the remaining bits.
+  to_return <- .finalize_bert_tokens(
+    tokenized_text = token_ids,
+    token_types = token_type_ids,
+    increment_index = increment_index
+  )
   return(
-    .finalize_bert_tokens(
-      tokenized_text = token_ids,
-      token_types = token_type_ids,
-      increment_index = increment_index,
-      simplify = simplify
-    )
+    structure(to_return,
```

Comment:
I feel like this should be inside `.finalize_bert_tokens` (thus the name), and then directly return the result of that call. Is there a reason you put it here?

Reply:
I thought about putting it there, but this... felt better? I don't have a good argument for why, though. :)

Reply:
...but I think you're right, because sometimes we return earlier than this! ok, I'll change it.

Reply:
This should be fixed now. Back to you!